packages/node-firebird-driver/README.md

```diff
@@ -18,6 +18,31 @@ There is also plan to create a socket-based (pure Node.js) driver.
 
 yarn add node-firebird-driver
 
+## Contributing
+- Clone this repo
+- `lerna bootstrap` (lerna must installed globally)
```

Comment:
lerna must not (and I don't recommend) be installed globally. The steps I use and recommend is:
```
yarn
yarn bootstrap
yarn test
```

Reply:
Ok. I'll be updating it to match that .